.github/workflows/shifty-build.yml

```diff
@@ -0,0 +1,43 @@
+name: Build Webapp
+
+on:
```

Comment:
We are already triggering this flow in the `build.yml` file - do we need to trigger it again here?